core/src/main/java/com/microsoft/applicationinsights/internal/heartbeat/HeartbeatDefaultPayload.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.applicationinsights.internal.heartbeat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class HeartbeatDefaultPayload {
```

Comment:
I would think this class is the Payload, but really its delegating things to PayloadProviders. Consider renaming.

Reply:
It is infact a container of default heartbeat providers and I think should be fine.